From 60a58ef46c1b597245bd50a21ca14c62902290a1 Mon Sep 17 00:00:00 2001
From: Alan Tull <atull@kernel.org>
Date: Tue, 19 Jun 2018 10:13:46 -0500
Subject: [PATCH 06/18] FogBugz #568760-1: fpga: stratix10 soc: comparison of
 unsigned against zero

commit  a947bdc08170f17b0478ad7ae23cb7aa00eab275 from
https://github.com/altera-opensource/linux-socfpga.git

Remedy the following Klocwork complaint:

drivers/fpga/stratix10-soc.c:330
CWARN.NOEFFECT.UCMP.LT (4:Review) Analyze Comparison of unsigned
value against 0 is always false

Signed-off-by: Alan Tull <atull@kernel.org>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/fpga/stratix10-soc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/fpga/stratix10-soc.c b/drivers/fpga/stratix10-soc.c
index 6664e22e2b06..740d44f21aae 100644
--- a/drivers/fpga/stratix10-soc.c
+++ b/drivers/fpga/stratix10-soc.c
@@ -302,7 +302,7 @@ static int s10_ops_write(struct fpga_manager *mgr, const char *buf,
 	struct s10_priv *priv = mgr->priv;
 	struct device *dev = priv->client.dev;
 	unsigned long timeout;
-	size_t sent = 0;
+	int sent = 0;
 	int ret = 0;
 	uint i;
 
-- 
2.17.1

